pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -128,6 +151,24 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 		// Added Queue back until no job in Queue.
 		queues.Push(queue)
 	}
+	// if target job exists, judge whether it can be inqueue or not
+	targetJob := util.Reservation.TargetJob
+	if targetJob != nil && targetJob.PodGroup.Status.Phase == scheduling.PodGroupPending && len(util.Reservation.LockedNodes) != 0 {
+		klog.V(3).Infof("Start to deal with Target Job")
+		minReq := api.NewResource(*util.Reservation.TargetJob.PodGroup.Spec.MinResources)
+		klog.V(3).Infof("Target Job request CPU: %f", minReq.MilliCPU)
+		lockedIdle := api.EmptyResource()
+		for _, lockNode := range util.Reservation.LockedNodes {
+			lockedIdle.Add(lockNode.Idle)
+		}
+		klog.V(3).Infof("locked idle CPU: %f", lockedIdle.MilliCPU)
+		idle = idle.Add(lockedIdle)
+		if minReq.LessEqual(idle) {
+			klog.V(3).Infof("Turn Target Job phase to Inqueue")
+			util.Reservation.TargetJob.PodGroup.Status.Phase = scheduling.PodGroupInqueue
+			ssn.Jobs[util.Reservation.TargetJob.UID] = util.Reservation.TargetJob
+		}
+	}
```

Comment:
have a question about this block of code: now that the locked node is determined, do we need to calculate `lockedNodesIdle` again? Why not use lockedNodesIdle calculated ahead?